### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by AnselemOdims**
# Steps Taken
- Created a new branch from the main branch
- Created a pushControl helper instance method
- Create a change helper instance method
- Created a tog helper instance method
- Rearranged logic for add and render helper instance methods 
- Included a display helper instance method
- Create the focusHandler and changeHandler instance methods
- Created a checkhandler instance method to handle checking of boxes
- Included display function in index.js, so list get display on load
- Updated README.md file

#Screenshot

## Before

todo

## After 

image

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -10,8 +10,78 @@ export default class Helper {
...
+    utils.tog('rem', 'd-none', child[1]);
+    utils.change(item.dataset.id, true);
+    child[1].addEventListener('click', (e) => {
+      utils.tog('rem', 'strike', child[2]);
+      utils.tog('add', 'd-none', e.target);
+      utils.tog('rem', 'd-none', item);
+      item.checked = false;
+      utils.change(item.dataset.id, false);
+    });
+  }
```
Comment (D4nielJ) [inline src/helpers.js]: - [ ] Let's make sure that when a task is toggled to completed or uncompleted, the state of the task is also stored in the LocalStorage. Right now, if I change a task to completed and try to reload the page, every task is set to uncompleted again.

Update: I checked the LocalStorage, and the state of each task is correctly being stored. So the bug occurs when you render the tasks.

image

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @AnselemOdims, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: 
- The list is stored correctly on the Local Storage
- UI it's on point
- Good job using JS modules!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://anselemodims.github.io/To-Do/dist/)
+[Live Demo Link](https://anselemodims.github.io/To-Do/src/)
 
 
 ## Getting Started
```
Comment (ArthurGC) [inline README.md:22]: - [OPTIONAL] It would be better to fix the broken MIT link in your README file. ♻️

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @AnselemOdims 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**